### ISSUE TITLE
[Tooltip] Tooltips in Menu items shown immediately after hovering

### ISSUE THREAD

**Issue by michaldudak** [mui/base-ui#522]
When a tooltip is placed on a menu item, it appears immediately when the item is hovered over, regardless of the `delay` setting. It is caused by browsers incorrectly applying the focus-visible state even though the menu was interacted with a mouse. Clicking on any other element before opening the menu makes the issue disappear.

Reproduction: https://codesandbox.io/p/sandbox/divine-meadow-xhsjpf

**Comment by onehanddev** [comment]
Hi @michaldudak, I’ve been looking into this bug. Would it be possible for me to take it on?

**Comment by michaldudak** [comment]
Yes, feel free to investigate it. The fix might need to be applied in Floating UI, though.

**Comment by michaldudak** [comment]
This issue doesn't occur anymore.
